configure.ac

```
@@ -2082,6 +2082,34 @@ case $CC in
         fi
 esac
 
+dnl Historically, some of our code assumed that signed integer overflow
+dnl is defined behaviour via twos-complement.  Prepare some flags so that
+dnl we can mark the modules that still do so.
```

Perhaps something like this?

```suggestion
dnl is defined behaviour via twos-complement.
dnl Set STRICT_OVERFLOW_CFLAGS and NO_STRICT_OVERFLOW_CFLAGS depending on compiler support.
dnl Pass the latter to modules that depend on such behaviour.
```